iphone/Classes/KrollContext.h

```diff
@@ -52,6 +52,13 @@
 #endif
 }
 
+// Worker properties
+bool TiWorker_setProperty(TiContextRef jsContext, TiObjectRef object, TiStringRef prop, TiValueRef value, TiValueRef *exception);
```

Comment:
So clearly if we're going to keep adding new types to our global, it'd be better to not clutter up KrollContext with all the methods. I believe in Windows, we point at the headers of all the proxy types and build the type in it's own definition/file. So it's not super-clean but it limits the exposure of the proxy type's implementation.

So basically we have assign the name "Worker" and the value is a call to something like TiWorker.Class() which sets up the JS type for JSC (it's parent type, the method callbacks for Initialize/Cosntructor, static functions, etc.)

That way TiWorker_setProperty/TiWorker_postMessage/TiWorker_terminate would move over to TiWorker.h/m